rad/src/script.rs

```diff
@@ -110,6 +114,56 @@ fn errorify(kind: RadError) -> RadError {
     kind
 }
 
+pub fn create_radon_script(
+    filters: &[RADFilter],
+    reducer: u32,
+) -> Result<Vec<RadonCall>, RadError> {
+    let unknown_filter = |code| RadError::UnknownFilter { code };
+    let unknown_reducer = |code| RadError::UnknownReducer { code };
+
+    let mut radoncall_vec = vec![];
+    for filter in filters {
+        let filter_op = filter.op;
+        let rad_filter = RadonFilters::try_from(filter_op as u8)
```

Comment:
`x as u8` is dangerous because `257 as u8 == 1`. `u8::try_from(filter_op)`?
```suggestion
        let rad_filter = RadonFilters::try_from(u8::try_from(filter_op).map_err(|_| unknown_filter(filter_op))?)
```